test/remote/gateways/remote_credorax_test.rb

```diff
@@ -285,6 +285,30 @@ def test_failed_refund
     assert_equal 'Referred to transaction has not been found.', response.message
   end
 
+  def test_successful_referral_cft
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+
+    options = {referral_cft: true}
+    response = @gateway.refund(@amount, response.authorization, options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
+    # Confirm that the transaction type was `referral_cft`
+    assert_equal '34', response.params['O']
```

Comment:
For our current Credorax gateway integration, these tests were passing but were using the `credit` transaction type instead of the `referral_cft` transaction type. I added this line to ensure that the transactions are going through as intended, but these would fail for us if the tests were actually working.